src/groups.ts

```diff
@@ -89,8 +107,8 @@ export function getSelectedGroupIdForElement(
 }
 
 export function getNewGroupIdsForDuplication(
-  groupIds: GroupId[],
-  editingGroupId: GroupId | null,
+  groupIds: ExcalidrawElement["groupIds"],
```

Comment:
same comment as above for the rest of these references

Reply:
But here it does make sense. You want to pass in element's `groupIds` verbatim (retaining order), when duplicating.

Reply:
uh, yeah i think you're right. carry on :)